### ISSUE TITLE
No property entryTasks for DefaultExecutionPlan 

### ISSUE THREAD

**Issue by francescobragagna** [dorongold/gradle-task-tree#37]
I've added the plugins config to the main build.gradle of my project, but when I run  `gradlew build taskTree`  (or any other task I tried) the build fails after few seconds, 
```
Execution failed for task ':taskTree'.
> No such property: entryTasks for class: org.gradle.execution.plan.DefaultExecutionPlan
```

I've already tried to use --task-depth <number> flag, or configure the taskTree in build.gradle file.

I'm using
Gradle 6.0
Kotlin:       1.3.50
Groovy:       2.5.8
Ant:          Apache Ant(TM) version 1.10.7 compiled on September 1 2019
JVM:          1.8.0_172 (Oracle Corporation 25.172-b11)
OS:           Windows 10 10.0 amd64

**Comment by sergeiwp** [comment]
This is Gradle 6 specific

**Comment by ksa-real** [comment]
This is Gradle 6 specific

**Comment by francescobragagna** [comment]
means Gradle6 it's not supported yet, or what?

**Comment by ksa-real** [comment]
I'm not one of the devs.
Just echoing what you've said and confirming the plugin works for me on 5.x but not on 6.x. I guess internal Gradle API has changed and the plugin must be fixed.

**Comment by ksa-real** [comment]
https://github.com/gradle/gradle/commit/b68df2666270e2401ec6d2aadd1cc21db0aa8554#diff-756e465001dc6712187ece0fabd15007
The diff where `entryTasks` were replaced with `entryNodes`.

**Comment by dorongold** [comment]
@ksa-real thanks for the fix!
Sorry for the long delay with merging this.
Gradle 6 compatibility is now released in version 1.5 of this plugin
34b1db8ea974a72fa2342135a2d5f7db894e3085
